robottelo/cli/factory.py

```diff
@@ -612,6 +606,8 @@ def make_location(options=None):
         --realms REALM_NAMES                        Realm names
                                                     Comma separated list of
                                                     values.
+        --provisioning-template[s|-ids]             Provisioning template Names/Ids
```

Comment:
These changes are weird a bit as this is the new way how this is written in help messages.